azure/packages/azure-client/package.json

```diff
@@ -81,7 +81,7 @@
 		"rimraf": "^4.4.0",
 		"start-server-and-test": "^1.11.7",
 		"typescript": "~4.5.5",
-		"uuid": "^8.3.1"
+		"uuid": "^9.0.0"
```

Comment:
Bumping 'uuid' to >= 9.0.0 fixes the issue described here:
https://stackoverflow.com/questions/73203367/jest-syntaxerror-unexpected-token-export-with-uuid-library